pkg/cloudinit/cloudconfig.go

```diff
@@ -18,9 +18,11 @@ import (
 )
 
 type CloudConfig struct {
-	Apt            Apt                `yaml:"apt,omitempty"`
-	Packages       []string           `yaml:"packages,omitempty"`
-	Runcmd         [][]string         `yaml:"runcmd,omitempty"`
+	Apt           Apt      `yaml:"apt,omitempty"`
+	Packages      []string `yaml:"packages,omitempty"`
+	PackageUpdate bool     `yaml:"package_update,omitempty"`
+	// Runcmd could be either []string or string
```

Comment:
why don't we just use `Runcmd: [][]string`? we can just use one method in archon.